Jenkinsfile

```diff
@@ -678,6 +678,58 @@ EXIT /b 1
           }
         }
 
+        stage('upload-artifacts-stable') {
```

Comment:
Stage  `upload-artifacts` will also upload to stable/

Reply:
Not with the changes from this pr, right?

Reply:
Oh yes. If `params.STABLE` is true this stage will rune and upload to `stable/`.
The stage above will use `DEPLOYMENT_PREFIX = getDeploymentPrefix()` and get `nightly/`